language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/common/utils/CommonUtil.java

```diff
@@ -303,15 +304,15 @@ private static String getDefaultValueForType(TypeSymbol bType, int depth) {
                 break;
             case OBJECT:
                 if (depth > MAX_DEPTH) {
-                    return "";
+                    return Optional.of("");
```

Comment:
I think we should send `new ()` in this case as we have done for record type, WDYT?

Reply:
In record types we return a `{}` to denote a complex record beyond level 1 and it's inserted after `Fieldname : ` (line 297). Hence it is necessary. But in the case of an object, `new()` is separately added in line 317/ 319 to the empty string returned from here. Returning `new ()` here explicitly will result in `new(new())`. Hence it is not necessary.

Reply:
Makes sense. Shall we do it? @TharushiJay

Reply:
Sorry, missed @TharushiJay's reply. If that's the case, let's let this be as it is.